owncloudApp/src/main/res/xml/settings_video_uploads.xml

```diff
@@ -45,7 +52,6 @@
         app:defaultValue="NOTHING"
         app:dialogTitle="@string/prefs_camera_upload_behaviour_dialog_title"
         app:iconSpaceReserved="false"
-        app:negativeButtonText=""
```

Comment:
This is necessary! Otherwise, an ugly "Cancel" button will appear in the AlertDialog. I think it looks better without it, like in Camera uploads screen 😄